hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HAUtils.java

```diff
@@ -385,9 +386,11 @@ public static List<String> buildCAList(CertificateClient certClient,
     long waitDuration =
         configuration.getTimeDuration(OZONE_SCM_CA_LIST_RETRY_INTERVAL,
             OZONE_SCM_CA_LIST_RETRY_INTERVAL_DEFAULT, TimeUnit.SECONDS);
+    SecurityConfig config = new SecurityConfig(configuration);
```

Comment:
Do we need to create this SecurityConfig object here, or can we create just the codec instance instead?